toolchain/parse/typed_nodes.h

```diff
@@ -312,6 +312,20 @@ struct FunctionDefinition {
   llvm::SmallVector<AnyStatementId> body;
 };
 
+using BuiltinFunctionDefinitionStart =
+    FunctionSignature<NodeKind::BuiltinFunctionDefinitionStart,
+                      NodeCategory::None>;
+using BuiltinName = LeafNode<NodeKind::BuiltinName>;
+
+// A builtin function definition: `fn F() -> i32 = "builtin-name";`
```

Comment:
```suggestion
// A builtin function definition: `fn F() -> i32 = "builtin.name";`
```

This seems to have fallen out of sync?

Reply:
This is just a placeholder to demonstrate that a string literal goes there. `"builtin.name"` might suggest that builtin names start with `builtin.`, which is not the current intent. Switched to a space instead; does that work for you?